Description: Link vlc with the C++ standard library
 See the corresponding vlc-cache-gen patch for more details.
Author: Benjamin Drung <bdrung@debian.org>
Bug-Debian: http://bugs.debian.org/661819

--- a/bin/Makefile.am
+++ b/bin/Makefile.am
@@ -16,7 +16,7 @@
 if !HAVE_WIN32
 if !HAVE_WINCE
 bin_PROGRAMS += vlc-wrapper
-vlc_SOURCES = vlc.c override.c
+vlc_SOURCES = vlc.c override.c dummy.cpp
 endif
 endif
 
